arch/csr/Smcntrpmf/mcyclecfgh.yaml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+$schema: csr_schema.json#
+kind: csr
+name: mcyclecfgh
+long_name: Machine Cycle Counter Configuration High
+address: 0x721
+priv_mode: M
+length: 32
+definedBy: Smcntrpmf
+description: |
+  Upper 32 bits of the 64-bit `mcyclecfg` CSR, used for RV32 systems to access
+  the privilege mode filtering inhibit bits.
+
+fields:
+  MINH:
+    location: 30
```

Comment:
I think all of these fields need to be defined as aliases of their equivalents in `mcyclecfg`. Here, for example:
```
  alias: mcyclecfg.MINH
```

Reply:
Related, but more general question for @dhower-qc if a field is defined as an "alias" of another field, do the rest of the attributes (type, definedBy, description, etc.) still need to be defined?